package.json

```diff
@@ -6,7 +6,9 @@
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server",
-    "start:abikit": "yarn build && ./digabi-koe-ohje-dev.sh",
+    "start:abikit": "npm-run-all build -p webpack-watch abikit-browser ",
+    "webpack-watch": "webpack --watch",
+    "abikit-browser": "./digabi-koe-ohje-dev.sh",
```

Comment:
Olikos tähän joku ihmeempi syy, miksi toi digabi-koe-ohje-dev.sh ei vois ottaa yhteyttä webpack-dev-serverin HTTP-palvelimeen `file://…/…/nönnönöö.html`-osoitteenn sijasta? Silloin sais varmaan live-päivityksen toimimaan out-of-the-box.

Reply:
Revertoin tän ja laitoin yhdistämään localhostiin, eli nyt pitää ensin käynnistää toi webpack dev server ja sitten erikseen abikit. Mun mielestä tää on parempi näin, niin noita pystyy nyt testamaan samaan aikaan.

Hot module reload ei näyttänyt toimivan ainakaan suoraan tolla abikit browserilla.